### DISCUSSION TITLE
Duels Leaderboard: no games since a couple of days?

### DISCUSSION THREAD

**Discussion by thomasweitzel** [BattlesnakeOfficial/feedback#258] [Bug Reports]
![duel-leaderboard-stuck-2024-08-08](https://github.com/user-attachments/assets/ec4237bd-24ed-41df-8965-55261e4eb4ec)

There is no progress: Games in progress 0%. The leaderboards were stuck at around 5 days ago.

Is this just a "reporting" issue on the website, or are there no games going on?

**Comment by bvanvugt** [comment]
Games are running again! Thanks for raising the issue.
